File: Client/src/app/Models/VMQueryObject.ts
import { Pagination } from "./Pagination";
export class VMQueryObject implements Pagination {
    constructor() {
        this.EmployeeID = 0;
       this.FromDate=new  Date().toISOString().slice(0,10);
       this.ToDate=new  Date().toISOString().slice(0,10);
    
    }
    FromDate: string;
    ToDate: string;
    EmployeeID: number;
    Status: number;
    PageIndex: number;
    PageSize: number;
    VehicleID: number;
    DriverID: number;

    Name:string
}

File: NybSys.Models/ViewModels/VMQueryObject.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace NybSys.Models.ViewModels
{
  public  class VMQueryObject
    {
        public DateTime FromDate { get; set; }

        public DateTime ToDate { get; set; }

        public int Status { get; set; }

        public int EmployeeID { get; set; }
        public int PageIndex { get; set; }
        public int PageSize { get; set; }

    }
}
